nucypher/utilities/versioning.py

```diff
@@ -25,7 +26,7 @@ class Versioned(ABC):
 
     _PARTS = 2  # bytes
```

Comment:
Just noticed this comment doesn't make sense on this line.  Perhaps just move it down inline with `_BRAND_SIZE`.